data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/handler/ProbeRequestUtil.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.knative.eventing.kafka.broker.receiver.impl.handler;
+
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.HttpServerRequest;
+
+public class ProbeRequestUtil {
+
+  public static final String PROBE_HEADER_NAME = "K-Network-Probe";
+
+  public static final String PROBE_HEADER_VALUE = "probe";
+
+  public static final String PROBE_HASH_HEADER_NAME = "K-Network-Hash";
+
+  public static boolean isProbeRequest(final HttpServerRequest request) {
```

Comment:
perhaps `isKubeletProbeRequest` to match upstream - since this is k8s?

Reply:
No, this is not for kubelet probe, this is for control plane probe, maybe `isControlPlaneProbeRequest`?

Reply:
+1 on the new, more descriptive name